### DISCUSSION TITLE
Get new document id

### DISCUSSION THREAD

**Discussion by SteeveDumais** [OrchardCMS/OrchardCore#6965] [Q&A]
Is there a way to get a proper document id when adding a new Document from a database script.  I see the value is stored in the Identifiers table but after the script is run the yesSqlSession and the table values are different until the session is renewed.  Is there a call I can do to refresh the yesSQLSession

**Comment by Skrypt** [comment]
Please provide an example of what you are trying to do so that we understand better.

**Comment by sebastienros** [comment]
If your document has an `Id` property it will be updated with the correct value.

**Reply by Skrypt** [reply]
I think he means when he creates a content item with a SQL script. Then the content item might not be created properly in the ContentItemIndex table if he has problems retrieving it from code on runtime at the same time.

**Reply by SteeveDumais** [reply]
Yes that is exactly what I mean.  I want to update the ContentItemIndex with a script at run time.  The Identifiers table is used to determine what the next document is going to be but if I update the document with a script the runtime application does not use the new Identifiers  number.
